composer.json

```
@@ -6,51 +6,66 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.4",
-        "drupal/core": "^9.3",
+        "php": ">=8.0",
+        "drupal/core": "^9.4",
         "drupal/entity_browser": "^2.5",
         "drupal/inline_entity_form": "~1.0-rc12",
-        "drupal/media_avportal": "^1.0-beta13",
+        "drupal/media_avportal": "dev-OEL-2115",
         "drupal/embed": "^1.3",
         "drupal/file_link": "^2.0.4"
     },
     "require-dev": {
         "behat/behat": "^3.10",
+        "behat/mink-selenium2-driver": "^1.5",
         "composer/installers": "~1.5",
-        "drush/drush": "~10.3",
-        "drupal/json_field": "1.0-rc4",
+        "drupal/composite_reference": "2.2.x-dev",
+        "drupal/config_devel": "^1.7",
+        "drupal/core-composer-scaffold": "^9.4",
+        "drupal/ctools": "3.9.0",
+        "drupal/drupal-extension": "dev-master as 4.0",
+        "drupal/entity_reference_revisions": "^1.7",
+        "drupal/json_field": "1.0-rc4 || ^1.1",
         "drupaltest/behat-traits": "^0.3",
-        "drupal/core-composer-scaffold": "^9.3",
-        "drupal/core-dev": "^9.3",
-	    "drupal/ctools": "3.9.0",
-        "fabpot/goutte": "^3.3",
-        "guzzlehttp/guzzle": "~6.3",
-        "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "^2.0",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta5",
-        "phpspec/prophecy-phpunit": "^2",
-        "openeuropa/oe_link_lists": "~0.16",
-        "openeuropa/oe_webtools": "^1.19",
-        "openeuropa/oe_oembed": "~0.1",
-        "drupal/drupal-extension": "~4.0",
-        "nikic/php-parser": "^4.12.0",
+        "drush/drush": "^11.1",
+        "guzzlehttp/guzzle": "^7.0",
         "instaclick/php-webdriver": "^1.4.7",
-        "behat/mink-extension": "^2.3.1",
-        "drupal/config_devel": "^1.7",
-        "twig/twig": "^1.40 || ^2.12",
-        "drupal/composite_reference": "^1.0@alpha",
-        "drupal/entity_reference_revisions": "^1.7"
+        "masterminds/html5": "^2.7.6",
+        "mikey179/vfsstream": "^1.6.10",
+        "nikic/php-parser": "^4.12.0",
+        "openeuropa/behat-transformation-context": "dev-master",
+        "openeuropa/code-review": "2.x-dev",
+        "openeuropa/oe_link_lists": "dev-master",
+        "openeuropa/oe_oembed": "dev-OEL-2115",
+        "openeuropa/oe_webtools": "dev-master",
+        "openeuropa/task-runner-drupal-project-symlink": "1.x-dev",
+        "phpspec/prophecy-phpunit": "^2",
+        "symfony/css-selector": "^4.4.24",
+        "symfony/dom-crawler": "^4.4.44",
+        "symfony/http-foundation": "^4.4.30",
+        "symfony/phpunit-bridge": "^6.0",
+        "symfony/routing": "^4.4.30",
+        "symfony/validator": "^4.4.30",
+        "symfony/yaml": "^4.4.29",
+        "twig/twig": "^1.40 || ^2.12"
```

^1.40 are not used anymore. Lowest used 2.15 version.
```suggestion
        "twig/twig": "^2.15"
```